libs/crypto/keys_manager.hpp

```diff
@@ -32,6 +32,7 @@ namespace iroha {
      * @param account_name
      * @return nullopt if no keypair found locally, or verification failure
      */
+    virtual nonstd::optional<iroha::keypair_t> loadKeys(std::string pubpath, std::string privpath) = 0;
```

Comment:
i guess strings shouldn't be copied

Reply:
This code is temporal, keymanager impl will be removed soon.